File: 1512-好数对的数目.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        int lens=nums.size();
        int ans=0;
        for(int i=0;i<lens-1;i++)
        {
            for(int j=i+1;j<lens;j++)
            {
                if(nums[i]==nums[j])
                    ans++;
            }
        }
        return ans;
    }
};

File: 1512-好数对的数目.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        lens = len(nums)
        ans=0
        for i in range(lens-1):
            for j in range(i+1, lens):
                if nums[i] == nums[j]:
                    ans += 1
        return ans